commercetools-models/src/test/java/io/sphere/sdk/carts/commands/CartUpdateCommandIntegrationTest.java

```diff
@@ -443,7 +443,6 @@ public void setShippingMethod() throws Exception {
                         .isEqualTo(shippingMethod);
                 assertThat(cartWithShippingMethod.getShippingInfo().getShippingMethod().getObj().getTaxCategory().getObj())
                         .as("reference expansion taxCategory")
-                        .isEqualTo(cartWithShippingMethod.getShippingInfo().getTaxCategory().getObj())
```

Comment:
is that not working anymore?

Reply:
Good catch 👍 i tough a `cartWithShippingMethod.getShippingInfo().getTaxCategory()` should have a `ResourceIdentifier` as a return but i was wrong, actually the resource identifier should be only on the draft level

i would fix that.